test/parallel/test-tty-stdout-end.js

```diff
@@ -1,16 +1,15 @@
 'use strict';
 // Can't test this when 'make test' doesn't assign a tty to the stdout.
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var exceptionCaught = false;
-
-try {
+const shouldThrow = function() {
```

Comment:
There were no other functions defined in the file, so I did function expressions because it is what I tend to use. There is precedence for function expressions in `test/parallel` files. `test-http-parser.js` has 21 function expressions. Still, it was basically an arbitrary decision on my part and I can switch them to function declarations in this file if you or anyone else has a strong preference.